doc/Command-Reference.md

```diff
@@ -5051,6 +5054,7 @@ Optional argument "-p" specify a period (in seconds) with which to gather counte
   show interfaces counters fec-histogram [-i <interface_name>]
   show interfaces counters fec-stats
   show interfaces counters detailed <interface_name>
+  show interfaces counters trim [interface_name] [-p|--period <sec>] [-j|--json]
```

Comment:
How about clear interface counter clearing the trim counters? portstat -c

Reply:
@kperumalbfn please have a look at `test_clear_port_trim_counters`